Makefile

```diff
@@ -0,0 +1,24 @@
+FROM_IMAGE=$(shell grep "^FROM" Dockerfile | cut -f2 -d' ')
+ORG?=simonferquel
+PROJ=integration-kit
+TAG?=$(shell git rev-parse --short HEAD)
+
+image:
+	docker inspect $(FROM_IMAGE) > /dev/null || docker pull $(FROM_IMAGE)
+	docker build -f Dockerfile -t $(ORG)/$(PROJ):$(TAG) .
+
```

Comment:
I would prefer to have makefile not dependent on docker, and instead do things like `docker run <img> make test-all`. That would allow to build/test/vet without docker.

Also if we could have a `setup` target installing golint (and maybe other tools we might want at some future point) to help setup dev environment, would be nice